tests/ignite/metrics/test_precision.py

```diff
@@ -825,19 +825,13 @@ def update(engine, i):
             _test(average=True, n_epochs=2, metric_device=metric_device)
 
     if idist.get_world_size() > 1:
-        with pytest.warns(
-            RuntimeWarning,
-            match="Precision/Recall metrics do not work in distributed setting when "
-            "average=False and is_multilabel=True",
-        ):
-            pr = Precision(average=False, is_multilabel=True)
-
+        pr = Precision(average=False, is_multilabel=True)
```

Comment:
First of all, could you please fix the issue we spotted during the call with unused `metric_device` that should go to `Precision` definition (line 795).

Ultimately, we would like to enable the tests in the same way as `test_multilabel_input_NCHW` where we run `_test(average=True, ...)` and `_test(average=False, ...)`. In case of average False, there is no equivalent option in sklearn (as probably it does not make much sense neither). So, please, take a look how we test things in `test_multilabel_input_NCHW` and do same thing here. Then we can simply remove this dummy test below.


PS.: average=False and is_multilabel=True gives as a result for precision = multilabel precision per sample
```python
import torch
from ignite.metrics import Precision

pr = Precision(average=False, is_multilabel=True)

pr.update((torch.tensor(y_pred[:5, :]), torch.tensor(y_true[:5, :])))
pr.update((torch.tensor(y_pred[5:, :]), torch.tensor(y_true[5:, :])))
print(pr._true_positives, pr._positives, pr.compute(), pr.compute().mean().item())
> (tensor([2., 2., 2., 2., 2., 2., 1., 0.], dtype=torch.float64),
 tensor([2., 3., 2., 2., 2., 2., 1., 1.], dtype=torch.float64),
 tensor([1.0000, 0.6667, 1.0000, 1.0000, 1.0000, 1.0000, 1.0000, 0.0000],
        dtype=torch.float64),
 0.8333333333333333)
```